UPGRADE-7.0.md

```diff
@@ -6,3 +6,8 @@ release process, both versions will have the same features, but Symfony 7.0 won'
 To upgrade, make sure to resolve all deprecation notices.
 
 This file will be updated on the branch 7.0 for each deprecated feature that is removed.
+
+Validator
+---------
+
+ * Changed `$versions` parameter type of the `Uuid::__construct()` method from `array` to `array|int`.
```

Comment:
I don't think we need an entry in the upgrade file.

Reply:
Are you sure? Because this is actually a BC break, that's why I targeted 6.4 at first and changed to 7.0 afterward :thinking:

Reply:
I thought about it again, you are right. I updated the PR accordingly :+1: